lib/que/poller.rb

```diff
@@ -265,5 +263,12 @@ def cleanup(connection)
         SQL
       end
     end
+
+    private
+
+    def poll_satisfied?(priorities, jobs)
+      lowest_priority = priorities.keys.max
```

Comment:
This was just a first crack at this, can probably do something a little more robust to select the requests for the lowest priority workers…

Reply:
What's not robust about it? Looks like it's always an integer - [`any` has been replaced by nil](https://github.com/que-rb/que/blob/e51f6c1826f12ccf07018b1cee3c9adfb456b3f7/bin/command_line_interface.rb#L72-L73) and then [nil replaced by the priority maximum](https://github.com/que-rb/que/blob/e51f6c1826f12ccf07018b1cee3c9adfb456b3f7/lib/que/job_buffer.rb#L128).

Reply:
Oh! I missed that. It’s probably fine then.